VariationalBayes/test_variational_bayes.py

```diff
@@ -470,6 +472,29 @@ def test_GammaParam(self):
         str(vp)
         vp.dictval()
 
+    def test_DirichletParamVector(self):
+        d = 4
+        alpha = np.array([ 0.2, 1, 3, 0.7 ])
+
+        vp = DirichletParamVector('test', dim = d, min_alpha=0.0)
+
+        # Check setting.
+        vp.alpha.set(alpha)
+
+        # Check size.
+        free_par = vp.get_free()
+        self.assertEqual(len(free_par), vp.free_size())
+
+        # Check getting and free parameters.
+        vp.alpha.set(np.full(d, 1.))
+        vp.set_free(free_par)
+        np_test.assert_array_almost_equal(alpha, vp.alpha.get())
+
+        # Just make sure these run without error.
+        vp.names()
```

Comment:
Following the new and cleaner way of doing the tests, please just add the ```DirichletParam``` to ```test_methods_work``` and remove redundant tests.  (I should have removed redundant tests from the other parameters.  I thought I did, but I must have forgotten.  If you like, you can remove them now or I will on a later PR.)